var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -85,6 +85,7 @@ class Hwloc(AutotoolsPackage):
     depends_on('libxml2', when='+libxml2')
     depends_on('cairo', when='+cairo')
     depends_on('numactl', when='@:1.11.11 platform=linux')
+    depends_on('ncurses')
```

Comment:
```suggestion
    depends_on('ncurses')
    depends_on('ncurses ~termlib')
```
Currently hwloc does not detect `libtinfo.so`, so you have to disable `termlib`